gravity-forms/gw-advanced-merge-tags.php

```diff
@@ -506,6 +506,35 @@ public function handle_field_modifiers( $value, $input_id, $modifier, $field, $r
 						return rgar( $value_array, $index );
 					}
 					break;
+				case 'all_fields':
+					// Only to be applied for GPPA scenarios, where we may using dynamically populated entry ID data.
+					if ( ! is_callable( 'gp_populate_anything' ) || ! gp_populate_anything()->is_field_dynamically_populated( $field ) ) {
+						break;
+					}
+
+					// {all_fields} merge tag support needs to get the entry IDs from the raw value.
+					$entries = json_decode( $raw_value, true );
+					
+					// loop through each entry ID and get the {all_fields} output.
+					$output  = '';
+					foreach ( $entries as $entry_id ) {
+						$entry = GFAPI::get_entry( $entry_id );
+						$form  = GFAPI::get_form( $entry['form_id'] );
+
+						// Safety check: Ensure the entry belongs to the same form as the field.
+						if ( $field->{'gppa-choices-primary-property'} != $entry['form_id'] ) {
```

Comment:
For added security, I'm curious if you could somehow recreate this logic or extract the logic in GPPA into a method that could be re-used: https://github.com/gravitywiz/gp-populate-anything/blob/a924ffcc780cc4bb01c117a194c1fa674fda6193/class-gp-populate-anything.php#L2608-L2644

Alternatively, if it's only used in the context of admins, you could add a security disclaimer to this modifier saying that it should not be used in places where non-admins can see the output of this merge tag modifier.